src/app/components/upload-dialog/file-uploading-dialog.component.ts

```diff
@@ -183,9 +186,10 @@ export class FileUploadingDialogComponent implements OnInit, OnDestroy {
       500: 'APP.MESSAGES.UPLOAD.ERROR.500',
       504: 'APP.MESSAGES.UPLOAD.ERROR.504',
       403: 'APP.MESSAGES.UPLOAD.ERROR.403',
-      404: 'APP.MESSAGES.UPLOAD.ERROR.404'
+      404: 'APP.MESSAGES.UPLOAD.ERROR.404',
+      generic: 'APP.MESSAGES.UPLOAD.ERROR.GENERIC'
     };
 
-    return messages[status];
+    return messages[status || 'generic'];
```

Comment:
Might not work for some non-supported keys as it uses generic only when status is missing.
```suggestion
    return messages[status] || messages['generic'];
```